monitor/src/test/java/edu/asu/diging/monitor/core/service/impl/AppManagerTest.java

```diff
@@ -64,7 +68,10 @@ public void test_addApp_success() throws UnstorableObjectException {
         String id = "ID1";
         Mockito.when(dbConnection.generateId()).thenReturn(id);
         IApp app = new App();
-        managerToTest.addApp(app);
+        AppForm appForm = new AppForm();
+        appForm.setUsername("user");
+        appForm.setPassword("password");
+        managerToTest.addApp(app, appForm);
```

Comment:
there should be two tests here, one that adds an app with password and one without. If there is one, it needs to check that the password has been encoded.